homeassistant/components/alexa/handlers.py

```diff
@@ -924,11 +925,13 @@ async def async_api_set_thermostat_mode(
     context: ha.Context,
 ) -> AlexaResponse:
     """Process a set thermostat mode request."""
+    operation_list: list[str]
```

Comment:
This can stay a local variable, it's not used outside of the if statement

Reply:
It is just typed, not assigned, later on it is assigned on 2 different spots and needs annotation.